### ISSUE TITLE
[Clients] change var names and table column names refering to belonging, responsible and unit

### ISSUE THREAD

**Issue by gaetanHerve** [bgo-bioimagerie/platformmanager#500]
These names refer to old ec classes. Have been replaced by client and pricing.
It's really confusing and it can generate coding errors to have mixed names for the same concept.

to be replaced:

- "belonging" by "pricing"
- "responsible" or "resp" by "client" in most cases
- "unit" by "client" in most cases

**Comment by osallou** [comment]
changing columns has impacts on db... high risk!!

**Comment by gaetanHerve** [comment]
yes, would take some time to handle and test all cases... I send it to 2.6. Not prio, but would be cleaner...
